datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -3818,6 +3818,180 @@ DROP TABLE min_bool;
 # Min_Max End #
 #################
 
+
+
+#################
```

Comment:
I will also make another PR to update the AggregateFuzz test added by @Rachelint  in  https://github.com/apache/datafusion/pull/12667 to cover these scenarios as well